### DISCUSSION TITLE
networking/traffic-lossless-upgrade

### DISCUSSION THREAD

**Discussion by giscus** [imroc/tke-guide#12] [General]
# networking/traffic-lossless-upgrade

业务场景

https://imroc.cc/tke/networking/traffic-lossless-upgrade

**Comment by ermazi** [comment]
好麻烦, 云资源操作跟 ng ing pod 生命周期没有"默契"啊, 这成本都赶上我 vm 维护 ng了; 真不如注册监听到集群外 ng

**Reply by imroc** [reply]
是, 这也是没完全将这功能产品化之前的办法, 操作麻烦, 但在一些核心网关维护场景是有用的
